mbuild/formats/lammpsdata.py

```diff
@@ -137,10 +143,15 @@ def write_lammpsdata(structure, filename, atom_style='full',
         sigma_conversion_factor = 1
         epsilon_conversion_factor = 1
         mass_conversion_factor = 1
-
-    # Internally use nm
-    box = Box(lengths=np.array([0.1 * val for val in structure.box[0:3]]),
-              angles=structure.box[3:6])
+    # lammps does not require the box to be centered at any a specific origin
+    # min and max dimensions are therefore needed to write the file in a consistent way
+    # the parmed structure only stores the box length
+    if mins:
```

Comment:
Should we do some fool proof check and make sure that `maxs` are also provided?

Reply:
I added in a function that performs some more tests (checks if both mins and maxs exist, and checks to see if the length is equal to 3). Also added more warnings about this.